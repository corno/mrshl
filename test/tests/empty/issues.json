[
    [ "parser", "error", "expected the schema start (!) or root value", 1, 1, 1, 1],
    [ "structure", "error", "missing schema", null, null, null, null]
]