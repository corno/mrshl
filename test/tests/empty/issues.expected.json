[
	[
		"deserialization",
		"error",
		"expected the schema start (!) or root value",
		1,
		1,
		1,
		1
	],
	[
		"structure",
		"error",
		"missing (valid) schema",
		null,
		null,
		null,
		null
	]
]